samples/project-anfield/import_cluster_config.py

```diff
@@ -1009,6 +1117,11 @@ def create_protection_jobs():
                 job_name = imported_job_prefix + job_name + imported_job_suffix
                 protection_job.name = job_name
             environment = protection_job.environment
+
+            # During source registration we provide environment type as kHyperV
+            # and as kHyperVVSS during Protection Group related CRUD functions.
```

Comment:
There's SCVMM and and standalone host . If we get standalone make sure this exits gracefully

Reply:
Updated.